tests/test_run.py

```diff
@@ -1,8 +1,12 @@
 # to suppress plt.show()
```

Comment:
(not necessary for this PR but can you at least come up with plan + github issue) To run CI tests that cleanvision can still work on filepath dataset without the optional dependencies installed: `torchvision`, `datasets`

Reply:
Added an issue for this